src/services/school/model.js

```diff
@@ -63,7 +63,7 @@ const schoolSchema = new Schema(
 			default: '',
 			enum: ['', 'school', 'schoolGroup'],
 		},
-		officialSchoolNumber: { type: String },
+		officialSchoolNumber: { type: String, unique: true, index: true }, // TODO we  should ensure this
```

Comment:
this might be true only in some instances, like BRB. But on boss, we need to check if schools from different federal states can have same OfficialSchoolNumber